.github/CODEOWNERS

```diff
@@ -0,0 +1,105 @@
+# This file describes the maintainers for the main components. See
+# `dev/doc/MERGING.md`.
+
+/checker/         @barras @maximedenes
+
+# We need a secondary maintainer for clib
+/clib/            @ppedrot
+
+/doc/             @silene @maximedenes
+/man/             @silene @maximedenes
+
+/engine/          @ppedrot @aspiwack
+
+# We need a secondary maintainer for grammar
+/grammar/         @ppedrot
+
+/ide/             @ppedrot @gares
+
+# We need a maintainer for interp
+#/interp/
```

Comment:
This looks stuff for @herbelin (notations, intern/exern-alization, ...)

Reply:
And what about the secondary maintainer of that part?